src/deluge/io/midi/midi_follow.h

```diff
@@ -32,7 +32,7 @@ class ModelStackWithThreeMainThings;
 class ModelStackWithAutoParam;
 enum class MIDIMatchType;
 
-Clip* getSelectedClip(bool useActiveClip = false);
+Clip* getSelectedClip(bool useActiveClip = false, ModelStack* modelStack = nullptr);
```

Comment:
This call signature is misleading now - if you set useActiveClip to true then a modelstack is required, and a modelstack isn't needed if useActiveClip is false.

Use function overloading instead, so that you have getSelectedClip() and getSelectedClip(ModelStack*)